CruCentralCoast/Community/CommunityGroups/CommunityGroupsTVC.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CommunityGroupsTVC.swift
+//  CruCentralCoast
+//
+//  Created by Landon Gerrits on 5/16/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class CommunityGroupsTVC: UITableViewController {
```

Comment:
Delete this class since it isn't used anymore. We don't want it to enter the repo